samples/joystick.py

```diff
@@ -0,0 +1,130 @@
+try:
```

Comment:
@austin-schick Is there a better way to import pygame than this try/except?

Reply:
I think this is the best way to do it without modifying Desktop CMU Graphics more. But here are a couple ideas to make this cleaner:

* We could make `pygame_loader` import local pygame if we're not in zipfile mode. Then you could always run the `except` part of this, and it might make other parts of our code cleaner too
* We could expose whatever pygame `cmu_graphics` is using as `cmu_graphics.pygame`
* We could pass pygame as a parameter to `handlePygameEvent`